dependencies.gradle

```diff
@@ -18,7 +18,7 @@ ext {
     revActivation = '2.0.1'
     revApacheHttpComponentsClient5 = '5.2.1'
     revAwaitility = '3.1.6'
```

Comment:
Do we need to update the dependencies.lock files via running:
`./gradlew generateLock saveLock
`

Reply:
The nebula plugin has been removed from build.gradle. Is there something else that I should use?

Reply:
Removed the lock files.